examples/cpp/infer_single/infer.cpp

```diff
@@ -0,0 +1,266 @@
+/*******************************************************************************
```

Comment:
Please change the name of the folder and file, infer_single_<what>?

Reply:
Done.